src/Cake.SqlServer/Cake.SqlServer.nuspec

```
@@ -8,8 +8,9 @@
     <title>Cake.SqlServer</title>
     <description>Cake Build addon for executing some operations in MS Sql Server.</description>
     <summary>SqlServer tools addon for Cake Build.</summary>
-    <licenseUrl>https://github.com/AMVSoftware/Cake.SqlServer/blob/master/LICENSE</licenseUrl>
-    <projectUrl>https://github.com/AMVSoftware/Cake.SqlServer</projectUrl>
+    <licenseUrl>https://github.com/cake-contrib/Cake.SqlServer/blob/master/LICENSE</licenseUrl>

```

```suggestion
    <licenseUrl>https://github.com/cake-contrib/Cake.SqlServer/blob/master/LICENSE</licenseUrl>
    <license type="expression">MIT</license>
```

@AdaskoTheBeAsT You might want to consider a switch to [license expression](https://docs.microsoft.com/en-us/nuget/reference/nuspec#license) instead whist you're at it, given that [`licenseUrl` is deprecated](https://github.com/NuGet/Announcements/issues/32) - or can be done in a future version... Just a heads-up

---

@augustoproiete  thanks for heads up - changed